OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -195,6 +198,11 @@ public DefaultSpriteSequence(ModData modData, string tileSet, SpriteCache cache,
 				var flipY = LoadField(d, "FlipY", false);
 
 				Facings = LoadField(d, "Facings", 1);
+				InterpolatedFacings = LoadField(d, "InterpolatedFacings", -1);
+				if (InterpolatedFacings != -1 && !(InterpolatedFacings > 1 && InterpolatedFacings > Facings && InterpolatedFacings <= 1024
+					&& Exts.IsPowerOf2(InterpolatedFacings)))
```

Comment:
```suggestion
				if (InterpolatedFacings != -1 && (InterpolatedFacings <= 1 || InterpolatedFacings <= Facings || InterpolatedFacings > 1024 || !Exts.IsPowerOf2(InterpolatedFacings)))
```
Since only one condition is needed to fail, using ORs makes this easier to read Imo. (And fits the wording you use in the exception message.)

Reply:
Makes sense. The only point I would say is given the fairly narrow set of valid numbers (9), describing the statements that are True seems more intuitive to me at least, but happy to go with the change as it can easily be argued both ways.